source/contact.erb

```diff
@@ -0,0 +1,104 @@
+
+    transport_directions: "nearest stations"
+    station_and_parking:
+      - name: "liverpool street"
+        link: "https://tfl.gov.uk/tube/stop/940GZZLULVT/liverpool-street-underground-station"
+        icon: "tfl-icon__underground.png"
+      - name: "shoreditch highstreet"
+        link: "https://tfl.gov.uk/overground/stop/910GSHRDHST/shoreditch-high-street-rail-station"
+        icon: "tfl-icon__overground.png"
+
+  - name: "Cape Town"
+    address: |
+      303 Buchanan Chambers
+      Cnr Pearce & Warwick Street
+      7735
+      Claremont, SA
+    humanized_phone_number: "+27 21 671 4503"
+    phone_number: "+27216714503"
+    email: "hello@unboxed.co"
+    map_link: "http://goo.gl/maps/JkNC9"
+    map_image: "https://a.tiles.mapbox.com/v4/web-unboxed.oec60hbb/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1Ijoid2ViLXVuYm94ZWQiLCJhIjoiY2lpN2hneWlvMDA4YnYzbHg2NHFqbW9yaSJ9.J1ksUtXP8Xsb2piQUcWJfA"
+
+    transport_directions: "nearest parking and station"
+    station_and_parking:
+      - name: "grove avenue"
+        link: "https://www.parkme.com/map?lot=122336"
+        icon: "transport-icon__car.png"
+      - name: "claremont"
+        link: "http://www.capemetrorail.co.za/Route-Map.html"
+        icon: "transport-icon__train.png"
+---
+
+<header class="page-header">
+  <div class="page-header__container">
+    <h1 class="page-header__title">
+      <%= current_page.data.title %>
+    </h1>
+
+    <p class="page-header__subtitle">
+      <%= current_page.data.subtitle %>
+    </p>
+  </div>
+</header>
+
+<div class="container--contact-page">
+  <% current_page.data.offices.each do |office| %>
+    <div class="contact-page-tile">
+      <div class="contact-page-tile__container">
+        <div class="contact-page-tile__content">
+          <p class="contact-page-tile__title"><%= office.name %></p>
+          <address class="contact-page-address">
+            <%= simple_format office.address %>
+          </address>
+          <div class="contact-page-address__phone-number">
+            <a href="tel:<%= office.phone_number %>">
+            <i class="fa fa-phone contact-page-address__phone-icon"></i>
+              <%= office.humanized_phone_number %>
+            </a>
+          </div>
+          <div class="contact-page-address__email"><a href="mailto:<%= office.email %>">
+            <i class="fa fa-envelope contact-page-address__email-icon"></i>
+            <%= office.email %>
+          </a></div>
+          <div class="contact-page-address__map-link">
+            <a href="<%= office.map_link %>" target="_blank">
+            <i class="fa fa-map-marker fa-lg contact-page-address__map-icon"></i>
+              open in maps
+            </a>
+          </div>
+          <div class="contact-page-address__directions">
+            <%= office.transport_directions %>
+            <% office.station_and_parking.each do |place| %>
+             <p class="contact-page-address__directions-station">
```

Comment:
This seems more like a `list` of transportation modes, instead of a paragraph. Also we might need to change the class names to be more generic as Grove Avenue isn't a station